app/src/app/[locale]/health/page.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Page() {
```

Comment:
is the /health/ page an aspect of next.js that's built in to handle some particular functionality? I couldn't find it looking in the docs.

Reply:
My understanding is it's required by the infra template in order to know if the container is healthy, otherwise it spins up a new ECS task.